.appveyor.yml

```diff
@@ -0,0 +1,39 @@
+# -*- mode: yaml -*-
+
+environment:
+
+  matrix:
+    - CONFIG: win_c_compilervs2015python3.6
+      CONDA_INSTALL_LOCN: C:\Miniconda36-x64
+
+    - CONFIG: win_c_compilervs2015python3.7
+      CONDA_INSTALL_LOCN: C:\Miniconda36-x64
+
+
+# We always use a 64-bit machine, but can build x86 distributions
+# with the TARGET_ARCH variable.
+platform:
+    - x64
+
+install:
+    # If there is a newer build queued for the same PR, cancel this one.
+
+    # Cygwin's git breaks conda-build. (See https://github.com/conda-forge/conda-smithy-feedstock/pull/2.)
+    - cmd: rmdir C:\cygwin /s /q
+
+    # Add path, activate `conda` and update conda.
+    - cmd: call %CONDA_INSTALL_LOCN%\Scripts\activate.bat
+    - cmd: conda.exe update --yes --quiet conda
+
+    - cmd: set PYTHONUNBUFFERED=1
+
+    # Configure the VM.
+    # Tell conda we want an updated version of conda-forge-ci-setup and conda-build
+    - cmd: conda.exe config --add channels conda-forge
+    - cmd: conda.exe install -n root -c conda-forge --quiet --yes conda-forge-ci-setup=2 conda-build
```

Comment:
I  thought "root" was now called "base"?

If you've added the conad-forge channel above, why do you need the `-c conda-forge` here?

Reply:
Another effect from the copying